analyzer/src/main/kotlin/managers/Npm.kt

```diff
@@ -113,6 +113,8 @@ open class Npm(
 
     private val graphBuilder = DependencyGraphBuilder(NpmDependencyHandler(this))
 
+    private val npmViewCache = mutableMapOf<String, JsonNode>()
```

Comment:
IIRC, the metadata for the packages is retrieved in multiple threads in parallel. So, the cache needs to be a concurrent map then.